### ISSUE TITLE
Local Library list view, poor performance with many items

### ISSUE THREAD

**Issue by rkben** [nukeop/nuclear#1645]
**Platform:**
macOS Sonoma 14.5, arm64

**Nuclear version:**
git master: https://github.com/nukeop/nuclear/commit/022ba2b98302939222bbe7dabb397ef0934c3b51

**Description of the issue:**
No exact number when performance begins to dwindle, around 2.2k items it struggles to render and scroll.

**Comment by rkben** [comment]
Most likely duplicate of https://github.com/nukeop/nuclear/issues/1290 and/or https://github.com/nukeop/nuclear/issues/1451

**Comment by rkben** [comment]
A workaround until virtual scrolling is implemented could be to disable images in the list view. I can test if a patch supplied.

**Comment by nukeop** [comment]
Yep, these are all connected. There are 3 different views, and they all need to implement virtualization.

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
